### ISSUE TITLE
Sort by stars

### ISSUE THREAD

**Issue by simnalamburt** [cheeaun/repokemon#6]
Thank you for making amazing repository. Can you make a functionality to sort the repositories(pokemons) by their stars?

**Comment by cheeaun** [comment]
Yeap, that's in my plans 👍

**Comment by limhenry** [comment]
should be easy if using polymer: dom-repeat's sort function... hmm (just 3-4 lines of codes)

**Comment by qgustavor** [comment]
How about [those changes](https://gist.github.com/qgustavor/d292babd7074a930ca61ddc650ebe53f)? As this this page seems too simple to require any library/framework (even the 5kb performant [choo](https://github.com/yoshuawuyts/choo)) I wrote those using only pure javascript.

**Comment by cheeaun** [comment]
@qgustavor thanks! Your code is a good reference for me 👍

**Comment by styfle** [comment]
I used this code to find pokemon that have no repo or low stars:

```js
var namesAvailable = pokemon.filter(d => !d.repo || d.repo.stars < 2).map( d => d.name);
console.log(namesAvailable);
```

**Comment by cheeaun** [comment]
Fixed in ebe61fd2486baffce93c138f01c818608da9f83f
